azure/services/vnetpeerings/vnetpeerings_test.go

```diff
@@ -23,8 +23,10 @@ import (
 	"testing"
 
 	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/to"
```

Comment:
according to the comment in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3340#top

> I think we want to switch to the `"k8s.io/utils/pointer"` library

we should use the k8s pointer library here as well

Reply:
Fixed